website/content/en/docs/getting-started/getting-started-with-karpenter/scripts/step08-apply-helm-chart.sh

```diff
@@ -1,6 +1,8 @@
 # Logout of helm registry to perform an unauthenticated pull against the public ECR
 helm registry logout public.ecr.aws
 
+# Set AWS_REGION env variable and annotate Karpenter service account
```

Comment:
It's not clear if this is going to be specific enough to give enough detail to users about what they need to install Karpenter on Faragte

Reply:
I removed the comment and added full command in the new tab "With Fargate".